src/plugins/kibana_usage_collection/kibana.json

```diff
@@ -3,9 +3,6 @@
   "version": "kibana",
   "server": true,
   "ui": false,
-  "requiredBundles": [
```

Comment:
__note:__ mentioned this in the Slack some time ago, this dependency isn't needed.